docs/3.x.x/en/cli/CLI.md

```diff
@@ -11,7 +11,7 @@ Create a new project
 ```bash
 strapi new <name>
 
-options: [--dev|--dbclient=<dbclient> --dbhost=<dbhost> --dbport=<dbport> --dbname=<dbname> --dbusername=<dbusername> --dbpassword=<dbpassword>]
+options: [--dev|--dbclient=<dbclient> --dbhost=<dbhost> --dbport=<dbport> --dbname=<dbname> --dbusername=<dbusername> --dbpassword=<dbpassword> --dbssl=<dbssl> --dbauth=<dbauth>]
```

Comment:
Do you think a better notation would be `--dbssl=[dbssl] --dbauth=[dbauth]`? Or should I leave it like this for consistency?